java/adapter/avro/src/main/java/org/apache/arrow/AvroToArrowUtils.java

```diff
@@ -102,6 +102,10 @@
    *   <li>ARRAY --> ArrowType.List</li>
    *   <li>MAP --> ArrowType.Map</li>
    *   <li>FIXED --> ArrowType.FixedSizeBinary</li>
+   *   <li>NULL --> ArrowType.Null</li>
```

Comment:
please move this into an avro related PR (or as a standalone one.

Reply:
Hmm, removed, thanks!